pkg/convert/pot_test.go

```diff
@@ -0,0 +1,317 @@
+package convert
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.expect.digital/translate/pkg/model"
+	"golang.org/x/text/language"
+)
+
+func Test_ToPot(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name     string
+		input    model.Messages
+		expected []byte
+	}{
+		{
+			name: "When all values are provided",
+			input: model.Messages{
+				Language: language.English,
+				Messages: []model.Message{
+					{
+						ID:          "Hello, world!",
+						Message:     "Bonjour le monde!",
+						Description: "A simple greeting",
+						Fuzzy:       true,
+					},
+					{
+						ID:          "Goodbye!",
+						Message:     "Au revoir!",
+						Description: "A farewell",
+						Fuzzy:       true,
+					},
+				},
+			},
+			expected: []byte(`"Language: en
+#. A simple greeting
+#, fuzzy
+msgid "Hello, world!"
+msgstr "Bonjour le monde!"
+#. A farewell
```

Comment:
Can we please add an empty line between messages to improve readability by human beings?

```suggestion
msgstr "Bonjour le monde!"

#. A farewell
```